libraries/botframework-schema/src/teams/index.ts

```diff
@@ -1690,13 +1690,27 @@ export type Action = 'accept' | 'decline';
 
 /**
  * @interface
- * Specific details of a Teams meeting.
  */
-export interface MeetingDetails {
+interface MeetingDetailsBase {
```

Comment:
Note: These interfaces were originally introduced in the PR that added `getMeetingInfo()`. Per Teams, the two schemas are intentionally different in the ways depicted below and not likely to change.